bundles/org.openhab.binding.gpio/README.md

```diff
@@ -46,13 +46,102 @@ sudo systemctl enable pigpiod
 sudo systemctl start pigpiod
 ```
 
-In openHAB, set `host` to the address of the pi and the `port` to the port of pigpio (default: 8888).
+## General Configuration
 
-Note: If you are running Pigpio on same host as openHAB, then set host to **::1**.
+Binding general configuration options. If you do not see all options, ensure `Show Advanced` is selected.
+
+Host:
+
+```
+Set Host to the address of the Pi that gpiod is running on. Default is 127.0.0.1.
+Note: If you are running pigpiod on same host as openHAB, set the host to ::1 (IPV6) or 127.0.0.1 (IPV4).
```

Comment:
OpenHAB

Reply:
I see you left this one open. And i'm not sure what the proper convention shoud be, i see it used different everywhere. Even in this single file.

Reply:
Not sure, I really just modified the original README. I set the default value to '127.0.0.1' as most folks are still running IPV4 (and presumably, most folks do run gpiod on the same host as OpenHAB), but I did want a note in there for the IPV6 only folks.

What about:

`Set Host to the address of the Pi that gpiod is running on. If gpiod is running on the same host as OpenHAB, set the host to 127.0.0.1 (IPV4) or ::1 (IPV6). The default is 127.0.0.1 (IPV4).`

Reply:
I just realized you may have been referring to the difference of gpiod vs pigpiod vs pigpio, etc. I think it should be normalized to pigpiod as that is what the official documentation refers to it as [here](https://abyz.me.uk/rpi/pigpio/pigpiod.html). Though, I've seen it referred to a variety of things on forums and I am guilty of this myself.

I have modified to README to just refer to it as **pigpiod**

Reply:
Last oke good to me, but I’m no expert @nils-bauer may add something.